### PR TITLE
♻️ Port amp-story info dialog to 1.0

### PR DESCRIPTION
**Pull request by newmuis**
This ports the info dialog that was added to amp-story 0.1 in #15130 to amp-story 1.0.

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-info-dialog.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Action, StateProperty} from './amp-story-store-service';
+import {CSS} from '../../../build/amp-story-info-dialog-0.1.css';
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-info-dialog.js]: 1.0

File: extensions/amp-story/1.0/amp-story-info-dialog.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    this.element_ = null;
+
+    /** @private {?Element} */
+    this.innerContainerEl_ = null;
+
+    /** @private {boolean} */
+    this.isBuilt_ = false;
+
+    /** @private @const {!./localization.LocalizationService} */
+    this.localizationService_ = Services.localizationServiceV01(this.win_);
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-info-dialog.js]: ``Services.localizationService``

File: extensions/amp-story/1.0/amp-story-info-dialog.js [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+    /** @private @const {!./amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = Services.storyStoreServiceV01(this.win_);
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-info-dialog.js]: ``Services.storyStoreService``

File: extensions/amp-story/1.0/amp-story-info-dialog.js (hunk 1 above)
Comment (newmuis) [reply extensions/amp-story/1.0/amp-story-info-dialog.js]: 😊 

Thanks for catching these!

**Pull request merged**